src/main/java/com/example/demo/Controller/StudentController.java

```diff
@@ -1,11 +1,10 @@
 package com.example.demo.Controller;
 import com.example.demo.Service.StudentService;
 import com.example.demo.StudentModel.Student;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import io.swagger.v3.oas.annotations.tags.Tags;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports

We shouldn't use wildcard imports because they hinter visibility and we import more things than we need from that class.

These should all be single imports as before, I know that Intellij formatted these automatically, but we need to undo it.

Reply:
Fixed.